apps/communications/contacts/test/marionette/lib/contacts.js

```diff
@@ -386,10 +388,35 @@ Contacts.prototype = {
     }
   },
 
+  getContactFirstName: function() {
+    var ci = this.client.findElement(Contacts.Selectors.contactsIframe);
+    this.client.switchToFrame(ci);
+
+    var contactInContactList = this.client.findElement
+      (Contacts.Selectors.listContactFirst);
+    var contactName = contactInContactList.text();
+    return contactName;
+  },
+
   tapSave: function() {
     this.client.helper.waitForElement(Contacts.Selectors.formSave).click();
   },
 
+  createNewContactAndReturnToDialer: function() {
+    this.client.helper.waitForElement(Contacts.Selectors.formGivenName);
+    this.client.helper.fillInputField(Contacts.Selectors.formGivenName, 'Name');
+    this.client.helper.waitForElement(Contacts.Selectors.formSave).click();
+  
+    var Dialer = require('../../../../dialer/test/marionette/lib/dialer.js');
+    var dialer = new Dialer(this.client);
+    this.client.switchToFrame();
+    this.client.apps.switchToApp
```

Comment:
Nit: use `dialer.switchTo()`.

Contacts shouldn't be aware of the way to switch to the dialer, it should remain as an implementation detail. `dialer.switchTo()` knows these details.